### PR TITLE
Update ember-cli to v2.9.1

### PR DESCRIPTION
**Pull request by jrjohnson**
Includes some library updates.  Does NOT include the bump to Ember 2.9
as we are still seeing some issues with that version.

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 78b28abe7e69980f529ba7a5582074d8ba6dfe0a

https://deploy-preview-2271--ilios-frontend.netlify.com

**Comment by jrjohnson** [comment]
Woo doggy.  Lots or borken stuffs here.  Will iterate.

File: bower.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "name": "ilios",
   "dependencies": {
-    "ember": "~2.8.1",
-    "ember-cli-shims": "0.1.1",
+    "ember": "~2.8.2",
```
Comment (stopfstedt) [inline bower.json]: is ember 2.9 a 'no go'?

File: package.json [hunk 2]
```diff
@@ -64,7 +64,7 @@
     "ember-concurrency": "0.7.15",
     "ember-cp-validations": "3.1.0",
     "ember-cryptojs-shim": "^1.0.0",
-    "ember-data": "^2.8.0",
+    "ember-data": "^2.8.1",
```
Comment (stopfstedt) [inline package.json]: ember data 2.9 still a no-go?

File: bower.json (hunk 1 above)
Comment (jrjohnson) [inline bower.json]: Yes. Decided to bump to next week to not overload this release.

File: package.json (hunk 2 above)
Comment (jrjohnson) [inline package.json]: Holding here for now. Will update to 2.9 next week.

**Pull request merged**